### PR TITLE
19th 3108

### PR DESCRIPTION
**Pull request by Clasher370**
ajax requests

### CODE REVIEW

File: app/assets/javascripts/card.js.coffee [hunk 1]
```diff
@@ -0,0 +1,11 @@
+$(document).ready ->
+  $(".card").on("ajax:success", (e, data, status, xhr) ->
+    console.log e
```
Comment (X1ting) [inline app/assets/javascripts/card.js.coffee]: you can use console.log e,data,status,xhr 
and you really need all console logs?

File: app/controllers/flashcards_controller.rb [hunk 2]
```diff
@@ -1,11 +1,19 @@
...
-  def index
-    @card = SetCardOnIndex.call(user: current_user, session: session[:card]).card
-  end
+  before_action :set_card, only: :index
 
   def compare
-    check_card = SuperMemo.new(params[:id], params[:user_text], params[:timer]).algorithm
-    redirect_to root_path, notice: check_card
+    notice = SuperMemo.new(params[:id], params[:user_text], params[:timer]).algorithm
+    set_card
```
Comment (X1ting) [inline app/controllers/flashcards_controller.rb]: why you invoke it manually? You can add it to before_action

File: app/views/layouts/_notice.html.erb [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<% flash.each do |_, msg| %>
+    <%= content_tag(:div, msg, class: 'alert alert-info') %>
```
Comment (X1ting) [inline app/views/layouts/_notice.html.erb]: indentation

File: app/assets/javascripts/application.js [hunk 4]
```diff
@@ -12,5 +12,4 @@
 //
 //= require jquery
 //= require jquery_ujs
-//= require turbolinks
```
Comment (X1ting) [inline app/assets/javascripts/application.js]: You delete it from Gemfile?

File: app/assets/javascripts/card.js.coffee (hunk 1 above)
Comment (Clasher370) [reply app/assets/javascripts/card.js.coffee]: think, i don't need it at all.

File: app/controllers/flashcards_controller.rb (hunk 2 above)
Comment (Clasher370) [reply app/controllers/flashcards_controller.rb]: I was do it before. But got bug, instance load before check and i can take checked card.

**Pull request merged**